README.md

```diff
@@ -2,144 +2,56 @@
 
-In the GitHub workflow called to validate a pull-request, insert the `Setup Build Scan dump capture` step once in each job having steps invoking Maven.
+Insert the `Setup Maven` step once in each job having steps invoking Maven. Make sure to put the step before the Maven invocation.
 
 ```yaml
 name: PR Build
 jobs:
   build:  
-      - name: Setup Build Scan dump capture
+      - name: Setup Maven
         uses: gradle/github-actions/maven-setup@v1
       - name: Build with Maven
         run: ./mvnw clean package
 [...]
 ```
 
-Add a new GitHub workflow to publish the Build Scans® saved during the previous step
+### Implementation details
 
-```yaml
-name: Upload Build Scans
+The action enables two features:
+- Display a summary of the Maven builds as a GitHub workflow summary or as a pull-request comment
+- Capture unpublished Build Scan® data as a workflow artifact per job with prefix `build-scan-data-maven`, which can then be published in a dependent workflow.
 
-on:
-  workflow_run:
-    workflows: [ "PR Build" ]
-    types: [ completed ]
-
-jobs:
-
-  publish-build-scans:
-    runs-on: ubuntu-latest
-    permissions:
-      actions: write
-      pull-requests: write
-    steps:
-      - name: Setup Build Scan link capture
-        uses: gradle/github-actions/maven-setup@v1
-      - name: Publish Build Scans
-        uses: gradle/github-actions/maven-publish-build-scan@v1
-        with:
-          develocity-url: 'https://<MY_DEVELOCITY_URL>'
-          develocity-access-key: ${{ secrets.<DEVELOCITY_ACCESS_KEY> }}
-```
-
-_Note:_<br>
-Some parameters need to be adjusted here:
-- The workflow name (here `PR Build`) triggered when a pull-request is submitted
-- The Develocity URL (here `https://<MY_DEVELOCITY_URL>`)
-- The secret name holding the Develocity access key (here `<DEVELOCITY_ACCESS_KEY>`)
- 
-#### Implementation details
+#### Summary
 
-##### maven-setup
-
-The action addresses two use cases:
-- Save unpublished Build Scan® data as a workflow artifact per job with prefix `build-scan-data-maven`, which can then be published in a dependent workflow.
-- Capture links of Build Scan® published to Develocity, which can then be displayed as a pull-request comment
+By default, a summary will be added to the GitHub workflow calling the action (can be skipped with `skip-job-summary` is set to `true`):
 
-The _capture strategy_ can be customized:
-- `ALWAYS`: default behavior, capture will be attempted on each Maven invocation
-- `ON_FAILURE`: capture will be attempted only on failed builds
-- `ON_DEMAND`: capture will be attempted if `CAPTURE_BUILD_SCAN=true` in the environment
+![workflow](./doc/summary-workflow.png)
 
-The _capture_ can be _enabled_/_disabled_ separately:
-- `capture-unpublished-build-scans`: to disable unpublished Build Scan® capture
-- `capture-build-scan-links`: to disable Build Scan® link capture
+By default, the pull-request will be commented with the summary (can be skipped if `skip-pr-comment` is set to `true`):
```

Comment:
```suggestion
By default, a comment will be added to the pull-request with the summary (can be skipped if `skip-pr-comment` is set to `true`):
```

I like that we're adding this comment by default. I think it is the best user experience but it does differ from how the Gradle action works.